scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/util/PolicyUtil.java

```
@@ -18,7 +18,7 @@ public static String getPolicyJsonContent() {
 				jsonPolicyStr += tmp;
 			}
 		} catch (IOException e) {
-			e.printStackTrace();
+			return null;
```

Please throw an exception.